frontend/src/lib/k8s/resourceQuota.ts

```diff
@@ -42,35 +42,113 @@ class ResourceQuota extends makeKubeObject<KubeResourceQuota>('resourceQuota') {
 
   get requests(): string[] {
     const req: string[] = [];
-    Object.keys(this.spec.hard).forEach(key => {
-      if (key.startsWith('requests.')) {
-        req.push(`${key}: ${this.status.used[key]}/${this.spec.hard[key]}`);
-      }
-    });
+    this.spec.hard &&
+      Object.keys(this.spec.hard).forEach(key => {
+        if (key.startsWith('requests.')) {
+          req.push(
+            `${key}: ${this.normalizeUnit(key, this.status.used[key])}/${this.normalizeUnit(
+              key,
+              this.spec.hard[key]
+            )}`
+          );
+        }
+      });
     return req;
   }
 
   get limits(): string[] {
     const limits: string[] = [];
-    Object.keys(this.spec.hard).forEach(key => {
-      if (key.startsWith('limits.')) {
-        limits.push(`${key}: ${this.status.used[key]}/${this.spec.hard[key]}`);
-      }
-    });
+    this.spec.hard &&
+      Object.keys(this.spec.hard).forEach(key => {
+        if (key.startsWith('limits.')) {
+          limits.push(
+            `${key}: ${this.normalizeUnit(key, this.status.used[key])}/${this.normalizeUnit(
+              key,
+              this.spec.hard[key]
+            )}`
+          );
+        }
+      });
     return limits;
   }
 
   get resourceStats() {
     const stats: { name: string; hard: string; used: string }[] = [];
-    Object.keys(this.status.hard).forEach(key => {
-      stats.push({
-        name: key,
-        hard: this.status.hard[key],
-        used: this.status.used[key],
+    this.status.hard &&
+      Object.keys(this.status.hard).forEach(key => {
+        stats.push({
+          name: key,
+          hard: `${this.normalizeUnit(key, this.status.hard[key])} (${this.status.hard[key]})`,
+          used: `${this.normalizeUnit(key, this.status.used[key])} (${this.status.used[key]})`,
+        });
       });
-    });
     return stats;
   }
+
+  normalizeUnit(key: string, quantity: string) {
```

Comment:
I don't see kubectl normalizing the units, I am not sure if we should go ahead and add this. @joaquimrocha WDYT?

Reply:
If kubectl is not doing, my inclination is to also not do it here. Maybe we can add a setting in the future to turn this on, but I would like to keep the current behavior if it matches kubectl.

Reply:
I think we should keep the same with kubectl. Maybe add a setting in the future.

Reply:
I got cases from our partners. The main purpose they use GUI to check resource quota is they want to know the usage of CPU and Memory. 
Kubectl doesn't normalize this unit indeed, but customers don't care the raw output, they need a straightforward way to know what how much they have used. 
Here is raw output from Headlamp, they have to do the math by converting the unit one by one.  
![image](https://user-images.githubusercontent.com/13809749/234531598-8e70f368-4712-4a8b-999b-576c36db65a8.png)
Alos the same as resource quota list, the unnormalized data only bring inconvenient to users.
![image](https://user-images.githubusercontent.com/13809749/234532536-8019f46b-b40c-4d63-87fa-3c1d2bcd5150.png)